src/Ledger/Introduction.lagda

```diff
@@ -59,16 +62,18 @@ Conway \cite{cip1694} & Complete & Partial & Partial \\
 \subsection{A Note on Agda}
 
 This specification is written using the Agda programming language and
-proof assistant \cite{agda2023}. We have spent a lot of time on making
+proof assistant~\cite{agda2023}. We have spent a lot of time on making
```

Comment:
It is good practice to link citations and references to the word preceding them with a tilde to avoid having numbers or symbols as the first characters on a line, unless they are part of a list of items.

Reply:
I feel this citations should go in parenthesis